src/posts.en-CA.md

```diff
@@ -7,7 +7,7 @@ eleventyNavigation:
   parent: en-CA
   key: Posts
   order: 3
-permalink: "{% if lang != config.defaultLanguage %}/{{ config.languages[lang].slug }}{% endif %}/{{ translations[lang].posts }}/{% if pagination.pageNumber > 0 %}page/{{ pagination.pageNumber + 1 }}/{% endif %}"
+permalink: "{% if lang != config.defaultLanguage %}/{{ config.languages[lang].slug }}{% endif %}/{{ translations[lang].posts }}/{% if pagination.pageNumber > 0 %}page/{{ pagination.pageNumber | plus: 1 }}/{% endif %}"
```

Comment:
I'm unclear to me from the [11ty docs](https://www.11ty.dev/docs/pagination/#remapping-with-permalinks) if this format works in nunjucks and liquid or just liquid. Makes sense to use this syntax if it works in both places.

Reply:
It only works in Liquid; the other one only works in Markdown. I made this change because Trivet does not modify the default template language for Markdown files, which is Liquid (see: https://www.11ty.dev/docs/config/#default-template-engine-for-markdown-files)

Reply:
Probably worth highlighting in the docs. The error returned after setting `markdownTemplateEngine: "njk"` but not updating the pagination doesn't appear to be immediately obvious as to what the issue actually is.

```log
[start:eleventy] [11ty] Problem writing Eleventy templates: (more in DEBUG output)
[start:eleventy] [11ty] > Having trouble rendering md template ./src/posts.fr-CA.md
[start:eleventy] 
[start:eleventy] `TemplateContentRenderError` was thrown
[start:eleventy] [11ty] > (./src/posts.fr-CA.md) [Line 1, Column 187]
[start:eleventy]   expected variable end
[start:eleventy] 
[start:eleventy] `Template render error` was thrown:
[start:eleventy] [11ty]     Template render error: (./src/posts.fr-CA.md) [Line 1, Column 187]
[start:eleventy]       expected variable end
[start:eleventy]         at Object._prettifyError (/Users/jo/Documents/git/trivet/node_modules/nunjucks/src/lib.js:36:11)
[start:eleventy]         at Template.init (/Users/jo/Documents/git/trivet/node_modules/nunjucks/src/environment.js:511:19)
[start:eleventy]         at Template.Obj (/Users/jo/Documents/git/trivet/node_modules/nunjucks/src/object.js:62:15)
[start:eleventy]         at new Template (/Users/jo/Documents/git/trivet/node_modules/nunjucks/src/environment.js:478:18)
[start:eleventy]         at Nunjucks.compile (/Users/jo/Documents/git/trivet/node_modules/@11ty/eleventy/src/Engines/Nunjucks.js:479:14)
[start:eleventy]         at Markdown.compile (/Users/jo/Documents/git/trivet/node_modules/@11ty/eleventy/src/Engines/Markdown.js:63:28)
[start:eleventy]         at TemplateRender.getCompiledTemplate (/Users/jo/Documents/git/trivet/node_modules/@11ty/eleventy/src/TemplateRender.js:231:26)
[start:eleventy]         at Template.compile (/Users/jo/Documents/git/trivet/node_modules/@11ty/eleventy/src/TemplateContent.js:294:42)
[start:eleventy]         at async Template._render (/Users/jo/Documents/git/trivet/node_modules/@11ty/eleventy/src/TemplateContent.js:388:16)
[start:eleventy]         at async Template._getLink (/Users/jo/Documents/git/trivet/node_modules/@11ty/eleventy/src/Template.js:252:24)
[start:eleventy] [11ty] Copied 1 file / Wrote 0 files in 0.15 seconds (v1.0.0)
[start:eleventy] npm ERR! code ELIFECYCLE
[start:eleventy] npm ERR! errno 1
[start:eleventy] npm ERR! trivet@0.1.0-dev.0 dev: `cross-env NODE_ENV=development eleventy --serve`
[start:eleventy] npm ERR! Exit status 1
[start:eleventy] npm ERR! 
[start:eleventy] npm ERR! Failed at the trivet@0.1.0-dev.0 dev script.
[start:eleventy] npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
[start:eleventy] 
[start:eleventy] npm ERR! A complete log of this run can be found in:
[start:eleventy] npm ERR!     /Users/jo/.npm/_logs/2022-01-26T14_10_08_151Z-debug.log
[start:eleventy] ERROR: "dev" exited with 1.
[start:eleventy] npm ERR! code ELIFECYCLE
[start:eleventy] npm ERR! errno 1
[start:eleventy] npm ERR! trivet@0.1.0-dev.0 start:eleventy: `run-p dev cms`
[start:eleventy] npm ERR! Exit status 1
[start:eleventy] npm ERR! 
[start:eleventy] npm ERR! Failed at the trivet@0.1.0-dev.0 start:eleventy script.
[start:eleventy] npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
[start:eleventy] 
[start:eleventy] npm ERR! A complete log of this run can be found in:
[start:eleventy] npm ERR!     /Users/jo/.npm/_logs/2022-01-26T14_10_08_266Z-debug.log
[start:webpack ] ℹ Compiling Mix
ERROR: "start:eleventy" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! trivet@0.1.0-dev.0 start: `npm-run-all -l clean -p start:*`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the trivet@0.1.0-dev.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jo/.npm/_logs/2022-01-26T14_10_08_401Z-debug.log
```

Reply:
@jobara I'm not sure exactly how it should be highlighted in the docs given that there's no change from the Eleventy default configuration… how would you suggest it be documented?

Reply:
I'd highlight the fact that Liquid syntax is used for the pagination and that changing the default `markdownTemplateEngine` will require changes to those areas.

Reply:
Resolved in 7dedf80.